### ISSUE TITLE
Lockable signs, hold key to change

### ISSUE THREAD

**Issue by Zoombini50** [StardewModders/mod-ideas#510]
On my very first Stardew farm, before I discovered mods and hovering crop labels, I tried to differentiate my crop plots by putting up wooden signs. Problem: it proved really, really easy to accidentally change the sign. I almost never use signs _anywhere_ anymore for this reason! (Well, and because I've learned my way around and no longer need a sign with a piece of wood on it to mark the way to Robin's house, haha.)

It would be neat if you had to hold down a modifier key to be able to change what's on a sign, similar to [the locking crab pots mod.](https://www.nexusmods.com/stardewvalley/mods/4461)

**Comment by MindMeltMax** [comment]
[LockedSignsMod.zip](https://github.com/StardewModders/mod-ideas/files/5115339/LockedSignsMod.zip)
This about what you are looking for?

**Comment by Zoombini50** [comment]
Yes, that's perfect! Thanks! Sorry for taking so long to test it; life got away from me and I haven't played much.
